recipes-devtools/deepstream/deepstream-5.0_5.0.0-1.bb

```diff
@@ -0,0 +1,73 @@
+DESCRIPTION = "NVIDIA Deepstream SDK"
+LICENSE = "Proprietary"
+LIC_FILES_CHKSUM = "file://usr/share/doc/deepstream-5.0/copyright;md5=f635f9f375e764ce281a2070599e2457"
+
+SRC_URI="https://repo.download.nvidia.com/jetson/common/pool/main/d/deepstream-5.0/deepstream-5.0_5.0.0-1_arm64.deb;subdir=${BPN}"
```

Comment:
We have a bbclass that will compute the SRC_URI for the packages NVIDIA serves via its package feed, you can replace this line with
```
inherit l4t_deb_pkgfeed

SRC_COMMON_DEBS = "${BPN}_${PV}_arm64.deb;subdir=${BPN}"
```
and that way if the base URL of the package feed changes, we won't have to change all the recipes.